### DISCUSSION TITLE
Project 1 Part 3 Question

### DISCUSSION THREAD

**Discussion by jackseitz** [cooperrc/computational-mechanics#224] [Q&A]
I keep getting a negative value for the time of the death in part 3 of project 1. The value I'm getting is approx. -1.55, which I'm assuming can be interpreted as being -1.55 hours before 11am... But I'm not sure if this makes sense. Am I doing something wrong? Or does this make sense?

**Comment by TylerBerendsohn** [comment]
I believe that makes sense. Generally we are viewing time as going forward so time values input into finding information on this equation is positive, therefore you would be predicting that 1.55 hours before 11am is when the time of death is... Negative hence it is using time in the past rather then in the future.

**Comment by cooperrc** [comment]
Negative time makes sense if you're considering 11am as hour-0. If you're using the initial 85 $^oF$ at 11, then -1.55 would mean 12.55 hours before the intial time.
